### DISCUSSION TITLE
When will the Agent build?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5930] [Q&A] [answered]
Hi, i created some DG and configured some settings. everything is fine. 
But maybe i wan't to add maybe "Messaging" with xmpp oder ntfy, will this result in baking a new agent for the DG's?
And if not, when will the agents be build with new settings?

**Comment by si458** [comment, marked as answer]
I'm guessing when u say DG you mean device groups.
The messaging is just simply notifications to the user about device disconnects, connects, help requests, etc...
It has nothing to do with customising the meshagent
